docs/site/tutorials/todo-list/todo-list-tutorial-repository.md

```diff
@@ -87,68 +87,40 @@ export class TodoListRepository extends DefaultCrudRepository<
 
 ### Inclusion of Related Models
 
-To get the related `Todo` object for each `TodoList`, we have to override the
-`find` and `findById` functions.
+To get the related `Todo` objects for each `TodoList`, we can use register a
+custom
+[`InclusionResolver`](https://loopback.io/doc/en/lb4/apidocs.repository.inclusionresolver.html)
+in the `TodoList` repository.
 
-First add the following imports:
+First, add the following import:
 
 ```ts
-import {Filter, Options} from '@loopback/repository';
-import {TodoListWithRelations} from '../models';
+import {InclusionResolver} from '@loopback/repository`;
```

Comment:
problem with single quote and backtick.

![image](https://user-images.githubusercontent.com/6864736/64700737-7888bc80-d475-11e9-851c-cefc2232dafd.png)

Reply:
Good catch! Thanks!